apps/web/src/app/dashboard/[name]/overview/page.tsx

```diff
@@ -14,18 +14,40 @@ import { Toggle } from "~/components/ui/toggle";
 import projectMockData from "~/lib/mock/organizationOverview";
 import { getOwnOrganizations } from "~/server/actions/organization";
 
-export default function Dashboard() {
-  // TODO: replace mocks with db queries
-  const { data: _organizations } = useQuery({
-    queryKey: ["organizations"],
-    queryFn: () => getOwnOrganizations(),
+type DashboardOverviewProps = {
+  params: {
+    name: string;
+  };
+};
+
+
+export default function Dashboard({ params }: DashboardOverviewProps) {
+  const { data: organizations, isLoading } = useQuery({
+    queryKey: ["organizations", params.name],
+    queryFn: async () => {
+      const [organizations, err] = await getOwnOrganizations()
+      if (err !== null) {
+        console.error(err);
+      }
+
+      return organizations;
+    },
   });
 
-  const organizationFromPathname = usePathname().split("/").at(2);
+  const router = useRouter();
+
   const [listDisplay, setListDisplay] = useState(false);
 
+  useEffect(() => {
+    console.log(organizations);
+    if (!isLoading && (!organizations || !organizations.map((org) => org.name).includes(params.name))) {
```

Comment:
Couldn't this be simplified to:
```suggestion
    if (!isLoading && !organizations?.map((org) => org.name).includes(params.name)) {
```